wsgi/odyssy/announcement/models.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.db import models
+from django.core.exceptions import ValidationError
+
+# Create your models here.
+class Announcement(models.Model):
+    """ Model for Announcement
+    key         -- Primary Key
+    initDate    -- Initialization Date
+    finDate     -- Final Date
+    title       -- Title of the Announcement
+    description -- Description of the Announcement
+    """
+
+    def __str__(self):
+        return "#" + str(self.key) + " " + self.title
+
+    def clean(self):
```

Comment:
about __str__ or clean. I myself checked the save method and __str__ method. But it was suggested by prakash. And yes sir it is working perfectly fine.